supabase/config.toml

```diff
@@ -69,9 +69,9 @@ file_size_limit = "50MiB"
 enabled = true
 # The base URL of your website. Used as an allow-list for redirects and for constructing URLs used
 # in emails.
-site_url = "http://127.0.0.1:3000"
+site_url = "http://127.0.0.1:5173"
 # A list of *exact* URLs that auth providers are permitted to redirect to post authentication.
-additional_redirect_urls = ["https://127.0.0.1:3000"]
+additional_redirect_urls = ["https://127.0.0.1:5173/auth-callback"]
```

Comment:
I had issues after chaining 2 password resets, because the 2nd time the URL is `https://127.0.0.1:5173/` instead of `https://localhost:5173/`, and for some reason then Supabase redirects to `https://localhost:5173//auth-callback` instead of `https://localhost:5173/auth-callback` (notice the extra `/`). This leads to a Not Found screen.

Maybe this can all be fixed by changing the config to:

```toml
site_url = "http://localhost:5173"
# A list of *exact* URLs that auth providers are permitted to redirect to post authentication.
additional_redirect_urls = ["https://localhost:5173/auth-callback"]
```